component/background/RoomService.tsx

```diff
@@ -11,11 +12,16 @@ const config = {
 };
 firebase.initializeApp(config);
 
-class RoomService {
+const allRooms = firebase.database().ref("rooms");
+const allUsers = firebase.database().ref("users");
+
+const MESSAGE = "message";
+const USER = "user";
```

Comment:
They are constants.
Like final static String variable in java.
I put them to the top to organize them.